src/types.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  FieldErrors,
+  FieldName,
+  Message,
+  MultipleFieldErrors,
+  AsProps,
+  Assign,
+} from 'react-hook-form';
```

Comment:
@bluebill1049 Should I fix core lib to export types?

Reply:
i think we can duplicate those types here if they are not essential for users. what do you think?

Reply:
this component probably not going to be touched after release for a while.